src/postal/smtp.clj

```diff
@@ -26,15 +26,26 @@
         [postal.support :only [make-props]])
   (:import [javax.mail Transport Session]))
 
+(defn- send-single [transport ^javax.mail.Message jmsg]
+  (.sendMessage transport jmsg (.getAllRecipients jmsg))
+  {:code (.getLastReturnCode transport)
+   :error :SUCCESS
+   :message (.getLastServerResponse transport)})
+
+(defn- send-multiple [transport jmsgs]
+  (doseq [^javax.mail.Message jmsg jmsgs]
+    (.sendMessage transport jmsg (.getAllRecipients jmsg)))
```

Comment:
Why not reuse `send-single` here?

Reply:
Ah, yes, that's be fine. I'll change that.